File: DogYears/DogYears.py
#Dog Years Calculator
#The first two years of a dog's life count as 21 human years.

i = int(input("What's the age of the dog? "))
extraLife = i - 2

if i == 1:
    print("The dog is 10.5 years old in human years.")
elif i == 2:
    print("The dog is 21 years old in human years.")
else:
    extraYears = extraLife * 4
    print(f"The dog is {21 + extraYears} years old in human years")

File: シ++/Codecademy Course/7DogYearsProject/dogYears.cpp
#include <iostream>
int main()
{
    double dogYear;
    std::cout << "What is the age of the dog? ";
    std::cin >> dogYear;

    double extraLife = dogYear - 2;
    double extraYears = extraLife * 4;
    double humanYears;
    {
        if (dogYear == 1)
        {
            humanYears = 10.5;
        }

        else if (dogYear == 2)
        {
            humanYears = 21;
        }

        else
        {
            humanYears = 21 + extraYears;
        }

    }

    std::cout << "The dog's age in human years is " << humanYears << " years old";
}
